### ISSUE TITLE
this.Command["user"] might not always be set

### ISSUE THREAD

**Issue by briankassouf** [Imgur/incus#62]
We should error if user is not set on pushormessage call

https://github.com/Imgur/incus/blob/master/message.go#L126

**Comment by AdriVanHoudt** [comment]
does this mean you can only websocket to certain users atm?

**Comment by briankassouf** [comment]
No, it means the `pushormessage` feature needs a specific user. The basis of this feature is if the user is on a web browser, then incus will route the message to the browser. If they are not online on the browser, it will route the message to the user's phone.

**Comment by AdriVanHoudt** [comment]
ok I see, makes sense
